src/main/java/com/marklogic/mule/extension/ConnectionProvider.java

```diff
@@ -160,10 +162,8 @@ private void configureSSL(DatabaseClientBuilder builder) {
         try {
             builder.withSSLContext(tlsContextFactory.createSslContext());
         } catch (Exception e) {
-            throw new RuntimeException(String.format(
-                "Unable to create SSL context; cause: %s",
-                tlsContextFactory.getKeyStoreConfiguration().getPath(), e.getMessage()
-            ), e);
+            String message = String.format("Unable to create SSL context; cause: %s", e.getMessage());
+            throw new ModuleException(message, ErrorType.CONNECTION_ERROR, e);
```

Comment:
Sonar is picky about generic exceptions being used - ModuleException is Mule's preferred base class for runtime exceptions.